docs/src/main/asciidoc/spring-cloud-commons.adoc

```diff
@@ -1045,6 +1045,53 @@ For that functionality, it is useful to have the selected service instance (whic
 
 By default, the name of the cookie is `sc-lb-instance-id`. You can modify it by changing the value of the `spring.cloud.loadbalancer.instance-id-cookie-name` property.
 
+=== Transform the load-balanced HTTP request
+
+You can use the selected `ServiceInstance` to transform the load-balanced HTTP Request.
+
+For `RestTemplate`, you need to implement and define `LoadBalancerRequestTransformer` as follows:
+
+[source,java,indent=0]
+----
+	@Bean
+	public LoadBalancerRequestTransformer transformer() {
+		return new LoadBalancerRequestTransformer() {
+			@Override
+			public HttpRequest transformRequest(HttpRequest request, ServiceInstance instance) {
+				return new HttpRequestWrapper(request) {
+					@Override
+					public HttpHeaders getHeaders() {
+						HttpHeaders headers = new HttpHeaders();
+						headers.putAll(super.getHeaders());
+						headers.add("X-InstanceId", instance.getInstanceId());
+						return headers;
+					}
+				};
+			}
+		};
+	}
+----
+
+For `WebClient`, you need to implement and define `LoadBalancerClientRequestTransformer` as follows:
+
+[source,java,indent=0]
+----
+	@Bean
+	public LoadBalancerClientRequestTransformer transformer() {
+		return new LoadBalancerClientRequestTransformer() {
+			@Override
+			public ClientRequest transformRequest(ClientRequest request, ServiceInstance instance) {
+				return ClientRequest.from(request)
+						.header("X-InstanceId", instance.getInstanceId())
+						.build();
+			}
+		};
+	}
+----
+
+If multiple transformers are defined, they will be applied in the order of Bean.
```

Comment:
Change "will be" to "are". Change "order of Bean" to "the order in which Beans are defined".

Reply:
Fixed in https://github.com/spring-cloud/spring-cloud-commons/pull/874/commits/f404b226706872779d69d9bac40fa2a9b8c6a01c